go/vt/vtgate/vindexes/vschema.go

```diff
@@ -183,6 +184,22 @@ type ksJSON struct {
 	Error    string            `json:"error,omitempty"`
 }
 
+func (ks *KeyspaceSchema) findTable(
```

Comment:
Extracted from `(*VSchema).findTable` to make smaller, easier to read functions.